.circleci/config.yml

```diff
@@ -98,7 +98,10 @@ jobs:
           command: |
             export AWS_ACCESS_KEY_ID=${STAGING_AWS_ACCESS_KEY_ID}
             export AWS_SECRET_ACCESS_KEY=${STAGING_AWS_SECRET_ACCESS_KEY}
-            aws s3 sync --delete --acl public-read packages/docs/next-doc-site/build s3://${BUCKET}.datacamp-staging.com/
+            aws s3 rm s3://${BUCKET}.datacamp-staging.com/ --recursive
+            aws s3 sync packages/docs/next-doc-site/build s3://${BUCKET}.datacamp-staging.com/ --acl public-read --exclude "*.js" --exclude "*.svg" --exclude "*.png" --exclude "*.jpg" --exclude "*.woff" --exclude "*.woff2" --cache-control public,max-age=0,must-revalidate
+            aws s3 sync packages/docs/next-doc-site/build s3://${BUCKET}.datacamp-staging.com/ --acl public-read --exclude "*" --include "*.js" --cache-control public,max-age=31536000,immutable
+            aws s3 sync packages/docs/next-doc-site/build s3://${BUCKET}.datacamp-staging.com/ --acl public-read --exclude "*" --include "*.svg" --include "*.png" --include "*.jpg" --include "*.woff" --include "*.woff2" --cache-control public,max-age=31536000,immutable,no-transform
```

Comment:
Can we version all the resource URLs that get a long max-age (either through directory structure or a hash in the file name) so we can easily update them?

Reply:
+1

Reply:
NextJS is hashing all JS files / assets imported with Webpack. Since fonts are in placed in public directory they are not hashed by Webpack. I guess we can add some version suffix to them manually (like v1). Although not sure it is really needed.